.github/workflows/e2e-sync.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
         uses: helm/kind-action@v1.4.0
         with:
           cluster_name: kind
-          node_image: kindest/node:v1.24.15
+          node_image: kindest/node:v1.25.11
```

Comment:
`Kubernetes version v1.24.15 does not match >=1.25.0-0`

flux-2.1.0 min supported K8s version being v1.25,  looks like kindest/node:v1.25.12/13 aren't available yet in the docker registry.